### PR TITLE
Fix translation

### PR DESCRIPTION
**Pull request by FritzHoing**
## Description
- Proposed imports need to be imported as default for the localize wrapper to work as intended.
- Adds missing dynamic translation to the ScaleDenominator files.

Breaking:
- remove label property

## Do you introduce a breaking change?

- [x] Yes
- [ ] No
- [ ] I am unsure (no worries, we'll find out)

### CODE REVIEW

File: src/Component/ScaleDenominator/MaxScaleDenominator.tsx [hunk 1]
```diff
@@ -51,8 +58,9 @@ export interface MaxScaleDenominatorProps extends Partial<MaxScaleDenominatorDef
  * Input field for the maximum scale of a rule.
  */
 export const MaxScaleDenominator: React.FC<MaxScaleDenominatorProps> = ({
-  label = 'Max. Scale',
-  placeholder = 'Enter max. Scale (Optional)',
+  placeholder,
+  locale = en_US.ScaleDenominator,
+  label = locale.maxScaleDenominatorLabelText,
```
Comment (KaiVolland) [inline src/Component/ScaleDenominator/MaxScaleDenominator.tsx:63]: I think the extra `label` prop should be removed here and we should use `locale.maxScaleDenominatorLabelText` in the JSX element instead.

File: src/Component/ScaleDenominator/MaxScaleDenominator.tsx [hunk 2]
```diff
@@ -63,16 +71,19 @@ export const MaxScaleDenominator: React.FC<MaxScaleDenominatorProps> = ({
   }
 
   return (
-    <Form.Item className="gs-max-scaledenominator" label={label} colon={false} {...formItemLayout} >
+    <Form.Item className="gs-max-scaledenominator"
+      label={label}
+      colon={false}
+      {...formItemLayout} >
```
Comment (KaiVolland) [inline src/Component/ScaleDenominator/MaxScaleDenominator.tsx:77]:
```suggestion
      {...formItemLayout}
  >
```

File: src/Component/ScaleDenominator/MinScaleDenominator.tsx [hunk 3]
```diff
@@ -51,8 +58,9 @@ export interface MinScaleDenominatorProps extends Partial<MinScaleDenominatorDef
  * Input field for the minimum scale of a rule.
  */
 export const MinScaleDenominator: React.FC<MinScaleDenominatorProps> = ({
-  label = 'Min. Scale',
-  placeholder = 'Enter min. Scale (Optional)',
+  placeholder,
+  locale = en_US.ScaleDenominator,
+  label = locale.minScaleDenominatorLabelText,
```
Comment (KaiVolland) [inline src/Component/ScaleDenominator/MinScaleDenominator.tsx:63]: see above

File: src/Component/ScaleDenominator/MinScaleDenominator.tsx [hunk 4]
```diff
@@ -63,16 +71,22 @@ export const MinScaleDenominator: React.FC<MinScaleDenominatorProps> = ({
   }
 
   return (
-    <Form.Item className="gs-min-scaledenominator" label={label} colon={false} {...formItemLayout}>
+    <Form.Item
+      className="gs-min-scaledenominator"
+      label={label}
+      colon={false}
+      {...formItemLayout}>
```
Comment (KaiVolland) [inline src/Component/ScaleDenominator/MinScaleDenominator.tsx:78]:
```suggestion
      {...formItemLayout}
  >
```

**Review by KaiVolland — COMMENTED** [review]
Nice.
But as you're refactoring it already we might consider to remove the `label` prop completly as it is part of the locale.
This would leed to a breaking change but we are doing a bigger refactoring with #1998 anyway so that should be no problem.

**Comment by FritzHoing** [comment]
Thank you for the review @KaiVolland !

**Review by KaiVolland — APPROVED** [review]

File: src/Component/ScaleDenominator/MinScaleDenominator.tsx [hunk 5]
```diff
@@ -63,16 +68,23 @@ export const MinScaleDenominator: React.FC<MinScaleDenominatorProps> = ({
...
-        placeholder={placeholder}
+        placeholder={locale.minScaleDenominatorPlaceholderText}
         onChange={onChange}
       />
     </Form.Item>
   );
 };
 
-export default MinScaleDenominator;
+// export default MinScaleDenominator;
```
Comment (jansule) [inline src/Component/ScaleDenominator/MinScaleDenominator.tsx:88]: Minor: Please remove this line

**Review by jansule — CHANGES_REQUESTED** [review]
Very nice @FritzHoing. Just one remark

File: src/Component/ScaleDenominator/MinScaleDenominator.tsx (hunk 5 above)
Comment (FritzHoing) [reply src/Component/ScaleDenominator/MinScaleDenominator.tsx]: Good catch! Thank you @jansule

**Review by jansule — APPROVED** [review]

**Pull request merged**